Kernel/Tasks/Coredump.cpp

```diff
@@ -377,10 +377,35 @@ ErrorOr<void> Coredump::create_notes_metadata_data(auto& builder) const
     return {};
 }
 
+static ErrorOr<void> add_elf_note_header(KBufferBuilder& builder)
+{
+    // From the System V ABI spec, chapter 5 "PROGRAM LOADING AND DYNAMIC LINKING"
```

Comment:
Maybe link <https://www.sco.com/developers/gabi/latest/ch5.pheader.html#note_section> here? Or maybe even better the new gABI: <https://gabi.xinuos.com/elf/07-pheader.html#note-sections> (<https://groups.google.com/g/generic-abi/c/doY6WIIPqhU/>)?

Reply:
I have to admit that I didn't search for a long time, but the spec I found was only a PDF. I will add the link!